nntrainer/meson.build

```diff
@@ -17,6 +17,8 @@ nntrainer_headers = [
    meson.current_source_dir() / 'nntrainer_logger.h',
    meson.current_source_dir() / 'nntrainer_error.h',
    meson.current_source_dir() / 'app_context.h',
+   meson.current_source_dir() / 'context.h',
+   meson.current_source_dir() / 'engine.h',
```

Comment:
This whole block could be rewritten as:
```
nntrainer_headers = files(
   'nntrainer_log.h',
   'nntrainer_logger.h',
   'nntrainer_error.h',
   'app_context.h',
   'context.h',
   'engine.h',
)
```
meson `files()` function creates list of files referencing current_source_dir()` those are files that are checked for existance in current source dir and returns list of files, rather than paths that may or may not exist.

Reply:
Good!